src/components/Identity/createComponent.js

```diff
@@ -49,7 +51,20 @@ export default ({
           return consent
             .awaitConsent()
             .then(() => {
-              return ecid ? undefined : getIdentity(options.namespaces);
+              const getIdentityOptions = [];
+              if (!isNil(options) && !isEmptyObject(options)) {
+                getIdentityOptions.push(options.namespaces);
+
+                const { edgeConfigOverrides: localConfigOverrides } = options;
+                const overrides = {
+                  ...globalConfigOverrides,
+                  ...localConfigOverrides
```

Comment:
I'm hesitant to use `deepAssign` here because I want to keep the web sdk faily agnostic to the internal structure of the config overrides object. That way, if Konductor decides to support further configuration overrides, or the blackbird schema changes, then we don't have to make any Web SDK changes. using `deepAssign` to merge known objects would couple things more than I like. 
Thoughts?

Reply:
We talked offline and realized deepAssign is needed here so that the config overrides are deeply combined.